scrapers/utils.py

```diff
@@ -14,7 +17,7 @@
 
 
 @contextmanager
-def download_directory(scraper_filepath):
+def download_directory(scraper_filepath, root_directory=None):
```

Comment:
Let's be a bit more explicit (and always loosen the scope a bit later):

```suggestion
def download_directory(scraper_filepath, zip_root_directory=None):
```

Reply:
I'm confused about this because the "root_directory" represents the root directory for the zip folder as well as all other circular files in this context.

Reply:
We can keep this as root_directory for now